src/main/java/mat/client/shared/CQLLibraryResultTable.java

```diff
@@ -275,6 +278,30 @@ private SafeHtml getCQLLibraryNameColumnToolTip(CQLLibraryDataSetObject object)
 		}
 		return sb.toSafeHtml();
 	}
+
```

Comment:
General question @nicole-hunter @AndrewBird81 : Is there any specific reason to have duplicate code/methods? I've seen almost similar code in measure screens and library screens. if you see two new methods that I added for this PR are almost same.  we can have some utility class and move common code there

Reply:
I think when developing it had been decided they were different enough that we would leave them separate. Other than that we don't really have a big reason. However we need to do a cost/benefit of taking the time now to rework it as this code could potentially all be gone in the near future.

Reply:
got it, thanks!